docs/process/what-is-desktop.md

```diff
@@ -0,0 +1,55 @@
+# What is GitHub Desktop and who is it for?
+
+### Purpose
+As we triage issues, evaluate pull requests, and discuss new features, the GitHub Desktop team is 
+making decisions based on our vision for the GitHub Desktop product. The purpose of this document 
+is to transparently share our collective view of what GitHub Desktop is and who we’re optimizing 
+for when making decisions. Our goal is to be able to point back to this document to ensure there are 
+fewer surprises and that we’re consistent in our product decisions. This will likely evolve 
+over time as we continue to learn more about our users and the problem space.
+
+## 1. GitHub Desktop reduces frustration and makes Git and GitHub workflows* more approachable.
+
+Our goal with GitHub Desktop is to remove frustration, awkward interactions, and “oh no” moments 
+for as many people as possible. We aim to make common workflows* (defined at the end of this document) 
+so simple that beginner and experienced developers alike are productive in working with Git and GitHub. 
+Git can be intimidating, and we want GitHub Desktop to make interactions with Git easier and more 
+approachable. GitHub Desktop is not a replacement for the functionality of Git, but a tool to enable 
+you and your team to be more productive.
+
+## 2. GitHub Desktop extends GitHub to your local environment.
+
+GitHub Desktop is built and maintained by a team at GitHub and a group of amazing community contributors 
+as an open source application. It is intended primarily to extend the features of GitHub, not to be an 
+agnostic Git client or replicate the feature set of github.com. While we support very basic functionality 
+that will allow the app to function with other hosting providers, we prioritize work that allows the 
+end-to-end GitHub and GitHub Enterprise experience to shine and takes advantage of the fact that we can 
+closely integrate with GitHub features.
+
+## 3. GitHub Desktop prioritizes workflows for beginners and teams working collaboratively.
```

Comment:
Suggested tweak: `teams working collaboratively` -> `collaborative teams`

Reply:
👍 Thanks! Made that change.